src/transformers/image_transforms.py

```diff
@@ -257,3 +260,56 @@ def resize(
         resized_image = np.array(resized_image)
         resized_image = to_channel_dimension_format(resized_image, data_format)
     return resized_image
+
+
+def normalize(
+    image: np.ndarray,
+    mean: Union[float, Iterable[float]],
+    std: Union[float, Iterable[float]],
+    data_format: Optional[ChannelDimension] = None,
+) -> np.ndarray:
+    """
+    Normalizes `image` using the mean and standard deviation specified by `mean` and `std`.
+
+    image = (image - mean) / std
+
+    Args:
+        image (`np.ndarray`):
+            The image to normalize.
+        mean (`float` or `Iterable[float]`):
+            The mean to use for normalization.
+        std (`float` or `Iterable[float]`):
+            The standard deviation to use for normalization.
+        data_format (`ChannelDimension`, *optional*, defaults to `None`):
+            The channel dimension format of the output image. If `None`, will use the inferred format from the input.
+    """
+    if isinstance(image, PIL.Image.Image):
+        warnings.warn("PIL will not be supported as input in the next release. Please use numpy arrays instead.")
```

Comment:
```suggestion
        warnings.warn("PIL will not be supported as input in the next release. Please use numpy arrays instead.", FutureWarning)
```
Also, please be explicit about the version: is it 4.25.0 (since we're developing 4.24.0 right now) or 5?

Reply:
Good point. Do we have a rule of thumb for how many releases we normally support?

Reply:
Normally we only do breaking changes at major releases, but for this one we're making an exception with a short deprecation cycle. Just checked with Lysandre and we should support the deprecation for two minors and fully remove this in 4.26.0 if that makes sense.

Reply:
Yep, makes sense! I'll update the warning message.